### ISSUE TITLE
Multiverse-Portals-bug

### ISSUE THREAD

**Issue by NathanDMT** [Multiverse/Multiverse-Portals#652]
Hello,
I would like to share this error message, it was spam every second in my console logs

[11:31:18 INFO]: [Multiverse-Portals-Debug] The PlayerInteractEvent was already cancelled. Doing nothing.

If you have any solution to fix it, I would like :)

**Comment by RaydanOMGr** [comment]
I don't know for sure but that looks like a plugin conflict.

**Comment by zax71** [comment]
Yep, a plugin conflict would make sense. I’d recommend a binary search to find the plugin

If the issue is reproducible with only the latest versions of Multiverse Core, Multiverse Portals and PaperMC installed then please reproduce it on a fresh server, send instructions for doing so alongside full logs in https://mclo.gs

**Comment by benwoo1110** [comment]
That’s a debug message, there can be many reason why it got cancelled, something even during normal gameplay due to how various game mechanic works. This does not point to any issue with multiverse or any particular plugin. You can just stop the logging with `/mv debug 0` :)
